frontend/uddhar-frontend/src/authentication/services/validation.jsx

```
@@ -0,0 +1,51 @@
+export const validateForm = ({ email, password, setErrors }) => {
+  let valid = true;
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'valid' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)